FirebaseAuth/Tests/Sample/Sample/SettingsViewController.m

```diff
@@ -29,6 +29,7 @@
 #import <FirebaseAuth/FirebaseAuth.h>
 #import "StaticContentTableViewManager.h"
 #import "UIViewController+Alerts.h"
+#import "../../../../FirebaseCore/Sources/Private/FIRVersion.h"
```

Comment:
Can this go back to non-relative paths now that Core is keeping it's private headers, or did you want to keep it like this?

Reply:
Hmm, it's arguable since we shouldn't really depend on private headers in samples. Let's revisit after we decide on whether the API should become public.